### PR TITLE
CI: Use conda for 3.8 build

### LINKED ISSUE #29001: CI: Integrate 3.8 builds
3.8 uses Python from Travis-CI.

Once conda packages are built for 3.8 we want to use our usual system.

### PR DESCRIPTION
**Pull request by TomAugspurger**
Closes https://github.com/pandas-dev/pandas/issues/29001

### CODE REVIEW

**Comment by jbrockmendel** [comment]
nice

**Comment by jbrockmendel** [comment]
LGTM

**Review by datapythonista — APPROVED** [review]
lgtm

File: .travis.yml [hunk 1]
```diff
@@ -30,11 +30,9 @@ matrix:
       - python: 3.5
 
     include:
```
Comment (WillAyd) [inline .travis.yml]: Any reason for changing this to trusty? Isn’t that an older image?

**Review by WillAyd — COMMENTED** [review]
Looks good minor question that may not matter

File: .travis.yml (hunk 1 above)
Comment (TomAugspurger) [reply .travis.yml]: The database services weren't starting up with bionic. Haven't looked into the differences at all.

**Comment by jreback** [comment]
thanks @TomAugspurger

**Pull request merged**